R/use_extendr.R

```diff
@@ -200,9 +213,11 @@ throw_if_invalid_rust_name <- function(name, call = caller_env()) {
 use_rextendr_template <- function(template,
                                   save_as = template,
                                   data = list(),
-                                  quiet = getOption("usethis.quiet", FALSE)) {
+                                  quiet = FALSE) {
+
+  if (quiet) withr::local_options(list("cli.default_handler" = function(...) { }))
```

Comment:
You are doing this a lot. Can you please refactor this into a custom `with` function instead?

Reply:
What would this look like for you? `with_quiet(quiet)` ?
Also, should this use `with_sink()` instead of `cli.default_handler`?

Reply:
`with_quiet(quiet)` looks nice & can be added to the docs as a go-to solution for all future public API functions